api/Trace/Clock.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface Clock
+{
+    /**
+     * A combination of the Monotonic and Realtime Clocks
+     * Monotonic time value in the first slot, as it'll get accessed more frequently in duration calculations.
+     * @return array
+     */
+    public function moment(): array;
+    public function now(): int;
```

Comment:
Let me make sure I understand this. Calling `moment()` will return an array like `[$monotonic, $real]` where `$monotonic` comes from the monotonic clock and `$real` comes from the realtime clock, right? Calling `now()` will give you the value from a realtime clock too?

Reply:
That was my initial intention.  If it works with you I think we can roll with it (and it might be worth adding additional documentation around these design decisions).

Reply:
We also talked in our meeting about making this `Clock` call a singleton.